packages/core/content-releases/admin/src/components/ReleaseModal.tsx

```diff
@@ -30,14 +32,18 @@ export const ReleaseModal = ({
   isLoading = false,
 }: ReleaseModalProps) => {
   const { formatMessage } = useIntl();
+  const { pathname } = useLocation();
+  const isCreatingRelease = pathname === `/plugins/${pluginId}`;
 
   return (
     <ModalLayout onClose={handleClose} labelledBy="title">
       <ModalHeader>
         <Typography id="title" fontWeight="bold" textColor="neutral800">
           {formatMessage({
-            id: 'content-releases.modal.add-release-title',
-            defaultMessage: 'New release',
+            id: isCreatingRelease
+              ? 'content-releases.modal.add-release-title'
+              : 'content-releases.modal.edit-release-title',
+            defaultMessage: isCreatingRelease ? 'New release' : 'Edit release',
```

Comment:
Sorry I think my previous comment was not clear 😞 I was wondering if we could handle it similar to this:

https://github.com/strapi/strapi/blob/cf91a2e72ad870f314927b51d0df7f85d88f7046/packages/core/content-releases/admin/src/translations/en.json#L11